packages/kit/src/core/adapt/prerender/fixtures/large-page/input.html

```diff
@@ -1732,7 +1732,7 @@ <h3 id="modules-$app-stores">$app/stores</h3>
 						<ul>
 							<li>
 								<code>getStores</code> is a convenience function around <code>getContext</code> that
-								returns <code>{ navigating, page, session }</code>. This needs to be called at the
+								returns <code>{ navigating, page, session, updating }</code>. This needs to be called at the
```

Comment:
removing this change (see #3661)

```suggestion
								returns <code>{ navigating, page, session }</code>. This needs to be called at the
```